src/test/regress/expected/view_propagation.out

```diff
@@ -0,0 +1,320 @@
+-- Tests to check propagation of all view commands
```

Comment:
we should add one distributed view test to src/test/regress/expected/upgrade_post_11_before.out which covers upgrade to 11 script handling views nicely.